awx_collection/tests/integration/targets/tower_inventory_source_update/tasks/main.yml

```diff
@@ -84,8 +84,8 @@
 
     - name: Test Inventory Source Update for All Sources
       tower_inventory_source_update:
+        name: "{{ item.name }}"
         inventory: "{{ inv_name }}"
-        inventory_source: "{{ item.name }}"
```

Comment:
For this integration test playbook task, it would be nice to have a check to make sure that the previously-implemented param of `inventory_source` still works, and state what we're doing in the task name.  Something like:

```
    - name: Test Inventory Source Update for All Sources (using inventory_source param vs name)
      tower_inventory_source_update:
        inventory_source: "{{ item.name }}"
```

Reply:
Added!